WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2367,3 +2368,22 @@ PrefMailComposeRowValueWrapper                 = width:@PrefMailComposeRowValueW
 PrefMailBlockMessageCell                       = width:@PrefMailBlockMessageCellWidth@
 PrefMailComposeRowSelectWidget                 = min-width:@PrefRowMiniControlWidth@
 PrefMailComposeRowColorPickerWidget            = width:@PrefMailComposeRowColorPickerWidgetWidth@
+
+######################################
+# Preference- Import-Export interface
+######################################
+
+PrefImpExtPage                  = 
+PrefImpExpActionBtn             = display:inline-block;
+PrefImpExpChooseFolderBtn       = width:auto; position:relative; top:4px;
```

Comment:
To override width being applied by other rule viz. `ZHasText .ZButtonTable`, which was adding width of `60px`

Thanks